.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+---
+name: build
+on: [ push, pull_request ]
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
```

Comment:
This may be worth a debate. I think given the length of our builds and the fact a lot of things fail on one specific gemset, this might be worth enabling

Only a thought though, I don't have enough time to fully consider it. I'll accept whatever you decide with